### PR TITLE
Semantic ui react kitchensink

### PR DESCRIPTION
**Pull request by mcallisto**
As discussed, added a simpler `semantic-ui-react` demo.

I also propose to introduce this notion of _**kitchen sink**_ demo, as something that is not intended to port an existing (a _**replica**_?) demo, but to pack and show as many as possible features of a given library.

In this sense, the current `antd` demo is a _kitchen sink_, while the `react-router-dom` is a _replica_.

Notably _kitchen sink_ demos welcome contributions to add new features, while _replica_ demos should be fairly stable. I tried to express this in a short `readme.md` file.

### CODE REVIEW

**Comment by oyvindberg** [comment]
I like it! Can you please delete the old one and just keep this? 

Also a nice to have would be some more callbacks to demonstrate that things work if something makes sense to add

**Comment by mcallisto** [comment]
I will try also to add a Modal, so to have state and callbacks

File: semantic-ui-react-kitchensink/assets/index.html [hunk 1]
```diff
@@ -3,12 +3,12 @@
 <head>
     <meta charset="UTF-8">
     <title>Semantic-ui-react demo</title>
-    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.1/dist/semantic.min.css"
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css"
```
Comment (oyvindberg) [inline semantic-ui-react-kitchensink/assets/index.html:6]: can we pull the styling from node_modules instead?

File: semantic-ui-react-kitchensink/src/main/scala/demo/Main.scala [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package demo
+
+import org.scalajs.dom
+import slinky.web.ReactDOM
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+
+object Main {
```
Comment (oyvindberg) [inline semantic-ui-react-kitchensink/src/main/scala/demo/Main.scala:9]: not really for this PR, but would it be worthwhile to inline this and trim the demos to typically one file only?

**Review by oyvindberg — APPROVED** [review]

File: semantic-ui-react-kitchensink/assets/index.html (hunk 1 above)
Comment (mcallisto) [reply semantic-ui-react-kitchensink/assets/index.html:6]: You mean using https://www.npmjs.com/package/semantic-ui-css?

Comment (mcallisto) [reply semantic-ui-react-kitchensink/assets/index.html:6]: > can we pull the styling from node_modules instead?

Not sure how this is done.

File: semantic-ui-react-kitchensink/src/main/scala/demo/Main.scala (hunk 2 above)
Comment (mcallisto) [reply semantic-ui-react-kitchensink/src/main/scala/demo/Main.scala:9]: Not sure, if you are coming straight from Slinky repos, the distinction between `Main` and `App` is always there. Plus I would also say that having each component in a separate file is a good practice that we should encourage.

**Comment by oyvindberg** [comment]
This all looks great @mcallisto . I added you to the repo, so I think you can merge yourself now? Just do it whenever you're happy

**Comment by mcallisto** [comment]
> Just do it whenever you're happy

Done, with a squash. The CSS from `node-modules` can be enhanced later on.

**Pull request merged**